index.html

```diff
@@ -8,41 +8,249 @@
+              <li class="li-style">HTML</li>
+              <li class="li-style">CSS</li>
+              <li class="li-style">JavaScript</li>
+            </ul>
+          </div>
+          <button class="btn-enable">See project</button>
+        </article>
+      </aside>
+
+      <aside>
+        <article class="work-card-container">
+          <img src="assets/Work2.png" alt="first project" />
+          <h2>Multi-Post Stories</h2>
+          <div class="subtitle-card-container">
+            <h4>CANOPY</h4>
+            <ul class="ul-flex-features">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required
+          </p>
+          <div>
+            <ul class="ul-stack-flex">
+              <li class="li-style">HTML</li>
+              <li class="li-style">CSS</li>
+              <li class="li-style">JavaScript</li>
+            </ul>
+          </div>
+          <button class="btn-enable">See project</button>
+        </article>
+      </aside>
+
+      <aside>
+        <article class="work-card-container">
+          <img src="assets/Work3.png" alt="first project" />
+          <h2>Tonic</h2>
+          <div class="subtitle-card-container">
+            <h4>CANOPY</h4>
+            <ul class="ul-flex-features">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required
+          </p>
+          <div>
+            <ul class="ul-stack-flex">
+              <li class="li-style">HTML</li>
+              <li class="li-style">CSS</li>
+              <li class="li-style">JavaScript</li>
+            </ul>
+          </div>
+          <button class="btn-enable">See project</button>
+        </article>
+      </aside>
+
+      <aside>
+        <article class="work-card-container">
+          <img src="assets/Work4.png" alt="first project" />
+          <h2>Multi-Post Stories</h2>
+          <div class="subtitle-card-container">
+            <h4>CANOPY</h4>
+            <ul class="ul-flex-features">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div>
+            <ul class="ul-stack-flex">
+              <li class="li-style">HTML</li>
+              <li class="li-style">CSS</li>
+              <li class="li-style">JavaScript</li>
+            </ul>
+          </div>
+          <button class="btn-enable">See project</button>
+        </article>
+      </aside>
+    </section>
+
+    <section class="about">
+      <aside class="main-background-setup">
+        <h1>About Myself</h1>
```

Comment:
Kindly, I suggest you use `h2` instead of `h1` for sub-heading for good semantics and best practices. 😊 
- Also I suggest you stick to design as much as you can, including (colours, font, margin, and padding). please change the heading color to `#172B4D` instead of black.